src/components/Navigation.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Navbar, Nav } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+
+const Navigation = () => {
+  return (
+    <Navbar bg='dark' expand='lg' fixed='top'>
+      <Navbar.Brand><Link to='/' className='header-link'>Reactshoppe</Link></Navbar.Brand>
```

Comment:
I think I called mine "AngularShoppeDemo" in the title tag.  And then it displays "THE FRUIT SHOPPE" similar to current fruiteshoppe.  Do we want to continue with the "Fruit Shoppe" convention or replace with our respective frameworks?

Reply:
I'll adjust mine to match yours